src/main/java/stsjorbsmod/patches/CullDexPatch.java

```diff
@@ -0,0 +1,55 @@
+package stsjorbsmod.patches;
+
+import com.evacipated.cardcrawl.modthespire.lib.SpirePatch;
+import com.evacipated.cardcrawl.modthespire.lib.SpirePrefixPatch;
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.powers.*;
+import javassist.CannotCompileException;
+import javassist.expr.ExprEditor;
+import javassist.expr.FieldAccess;
+import javassist.expr.NewExpr;
+import stsjorbsmod.characters.Cull;
+
+@SpirePatch(
```

Comment:
I think the mechanics of the patch seem reasonable, but I would suggest putting it as a prefix patch on `update` (wrapped in a `if (duration == startingDuration)`) rather than an instrument patch on this specific CTOR; historically, there have been a few instances of the base game introducing new CTOR overloads (possibly changing which CTOR is the "true" base one you'd want this patch to apply to), but it's very unlikely that the signature of `AbstractPower.update()` will change
much more common for future changes to the base game to introduce new CTOR overloads than